.zenodo.json

```
@@ -0,0 +1,162 @@
+{
+    "creators": [
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Vay, Jean-Luc"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Almgren, Ann"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Amorim, L\u00edgia Diana"
```

@LDAmorim what is your ORCiD? :)
```suggestion
        "name": "Amorim, L\u00edgia Diana",
       "orcid": "..."
```

---

```suggestion
        "name": "Amorim, L\u00edgia Diana",
        "orcid": "0000-0002-1445-0032"
```

---

Thanks @ax3l !